egs/wsj/s5/steps/data/augment_data_dir.py

```diff
@@ -175,13 +176,15 @@ def main():
         new_wav = AugmentWav(utt, wav, dur, fg_snrs, bg_snrs, fg_noise_utts,
             bg_noise_utts, noise_wavs, noise_reco2dur, args.fg_interval,
             num_bg_noises)
-        new_utt = utt + "-" + args.utt_suffix
+        new_utt = utt + "_" + args.utt_suffix 
```

Comment:
I would err on the side of not changing these things unless there's a good reason.